src/extension.ts

```diff
@@ -82,7 +82,7 @@ function configurationChanged() {
 }
 
 export function activate(context: vscode.ExtensionContext): void {
-    setupLSP();
+    //setupLSP();
```

Comment:
I was getting an error starting the LSP. Was this disabled? Should this line still be active?

Reply:
This line is what starts the language server for Racket files. It should still be active, and you shouldn't release an update to the extension with it commented.

Reply:
uncommented!